### PR TITLE
Add Python to Shuffleboard docs

### PR DESCRIPTION
**Pull request by jasondaming**

### CODE REVIEW

File: source/docs/software/dashboards/shuffleboard/advanced-usage/shuffleboard-tuning-pid.rst [hunk 1]
```diff
@@ -69,3 +69,19 @@ The following example demonstrates how to create a button on your dashboard that
...
+
+  .. code-block:: python
+
+    from wpilib.shuffleboard import Shuffleboard
+
+    tab = Shuffleboard.getTab("Shooter")
+    shooterEnable = tab.add("Shooter Enable", false).getEntry()
+
+    # Command Example assumed to be in a PIDSubsystem
+    NetworkButton(shooterEnable).onTrue(InstantCommand(m_shooter.enable()))
```
Comment (sciencewhiz) [inline source/docs/software/dashboards/shuffleboard/advanced-usage/shuffleboard-tuning-pid.rst:81]: Out of curiosity, have you tried running this? I don't think I ever got NetworkButton to work. Obviously not a python issue

File: source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-displaying-data.rst [hunk 2]
```diff
@@ -13,18 +13,42 @@ Displaying values in normal operating mode (autonomous or teleop)
...
+         SmartDashboard.putNumber("Bridge Angle", bridgeTipper.getPosition());
+         SmartDashboard.putNumber("Swerve Angle", drivetrain.getSwerveAngle());
+         SmartDashboard.putNumber("Left Drive Encoder", drivetrain.getLeftEncoder());
+         SmartDashboard.putNumber("Right Drive Encoder", drivetrain.getRightEncoder());
+         SmartDashboard.putNumber("Turret Pot", turret.getCurrentAngle());
+         SmartDashboard.putNumber("Turret Pot Voltage", turret.getAverageVoltage());
+         SmartDashboard.putNumber("RPM", shooter.getRPM());
+      }
+
+   .. code-block:: cpp
```
Comment (sciencewhiz) [inline source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-displaying-data.rst:29]: Other places you changed cpp to c++

File: source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-displaying-data.rst [hunk 3]
```diff
@@ -13,18 +13,42 @@ Displaying values in normal operating mode (autonomous or teleop)
...
+
+      frc::SmartDashboard::PutBoolean("Bridge Limit", bridgeTipper.atBridge());
```
Comment (sciencewhiz) [inline source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-displaying-data.rst:31]: C++ convention would have AtBridge() (same with everything below)

File: source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-tour.rst [hunk 4]
```diff
@@ -39,13 +39,23 @@ You can start Shuffleboard in one of four ways:
...
+The easiest way to get data displayed on the dashboard is simply to use methods in the ``SmartDashboard`` class. For example to write a number to Shuffleboard write:
 
 .. tab-set-code::
 
    .. code-block:: java
 
-        SmartDashboard.putNumber("Joystick X value", joystick1.getX());
+      SmartDashboard.putNumber("Joystick X value", joystick1.getX());
+
+   .. code-block:: cpp
```
Comment (sciencewhiz) [inline source/docs/software/dashboards/shuffleboard/getting-started/shuffleboard-tour.rst:50]: another cpp

File: source/docs/software/dashboards/shuffleboard/layouts-with-code/retrieving-data.rst [hunk 5]
```diff
@@ -6,19 +6,39 @@ Unlike ``SmartDashboard.getNumber`` and friends, retrieving data from Shuffleboa
...
+         private DifferentialDrive robotDrive = ...;
+
+         public void drive(double left, double right) {
+         // Retrieve the maximum speed from the dashboard
+         double max = maxSpeed.getDouble(1.0);
+         robotDrive.tankDrive(left * max, right * max);
+         }
+      }
+
+   .. code-block:: python
```
Comment (sciencewhiz) [inline source/docs/software/dashboards/shuffleboard/layouts-with-code/retrieving-data.rst:24]: did you intentionally not add C++?

Comment (jasondaming) [reply source/docs/software/dashboards/shuffleboard/layouts-with-code/retrieving-data.rst:24]: I didn't add C++ cause I always mess it up.  Wouldn't mind inserting it though.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**